tests/test_benchmark.py

```diff
@@ -378,16 +418,15 @@ def test_nonbonded_interaction_group_potential(hi2fa_test_frames):
 
 
 def test_nonbonded_potential(hi2fa_test_frames):
-    pots, sys_params, frames, boxes, _ = hi2fa_test_frames
-    U_names = [type(U_fn).__name__ for U_fn in pots]
-    nb_idx = U_names.index("Nonbonded")
-    nonbonded_pot = pots[nb_idx]
-    assert isinstance(nonbonded_pot, Nonbonded)
+    pots, frames, boxes, _ = hi2fa_test_frames
+
+    nonbonded_pot = pots[-1]
```

Comment:
nit: probably safer not to assume an ordering here? Maybe
```suggestion
    nonbonded_pot = next(p for p in pots if isinstance(p, Nonbonded))
```

Reply:
good idea, willfix inline